### PR TITLE
Add tutorial for colorspace conversions

### PR DESCRIPTION
**Pull request by yongtang**
This PR adds a tutorial for colorspace conversions

Signed-off-by: Yong Tang <yong.tang.github@outlook.com>

### CODE REVIEW

**Comment by lamberta** [comment]
H @yongtang 

We're starting to [add CI](https://github.com/tensorflow/docs-l10n/blob/master/.github/workflows/ci.yaml#L8) for formatting and linting checks to docs repos. Still testing it out but would you mind giving it a try on this notebook? Thanks

```
$ python3 -m pip install -U --user git+https://github.com/tensorflow/docs

$ python3 -m tensorflow_docs.tools.nbfmt notebook.ipynb
$ python3 -m tensorflow_docs.tools.nblint --arg=repo:tensorflow/io notebook.ipynb
```

File: docs/tutorials/colorspace.ipynb [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "VSkY6UAxODOq",
+        "colab_type": "text"
+      },
+      "source": [
+        "### Download the sample image\n",
+        "\n",
+        "The image example used in this tutorial is from a downloadable screenshot of [Fallout 4](https://fallout.bethesda.net/en/games/fallout-4), available in the media section of the website. For convenience, a [copy](https://raw.githubusercontent.com/tensorflow/io/master/tests/test_image/fallout-4.jpg) has been saved on GitHub. The image sample could be replaced by any JPEG image for this tutorial.\n",
```
Comment (MarkDaoust) [inline docs/tutorials/colorspace.ipynb:155]: We do end up republishing the images so we always want to be careful about image copyright,. and it's best if we pull images from resources that we actually control.

So could you switch this to something from download.tensorflow.org/example_images? Any of these would work:

https://storage.googleapis.com/download.tensorflow.org/example_images/
* grace_hopper.jpg.
* 320px-Felis_catus-cat_on_snow.jpg
* Green_Sea_Turtle_grazing_seagrass.jpg
* San_Francisco_fireboat_showing_off.jpg
*  YellowLabradorLooking_new.jpg

**Comment by yongtang** [comment]
Thanks @lamberta @MarkDaoust. The PR has been updated with tutorial formatted with tensorflow_docs.tools.nbfmt and tensorflow_docs.tools.nblint.

The image has been replaced with 320px-Felis_catus-cat_on_snow.jpg as well.

On a side note. I have created another PR #1049  to setup GitHub CI and always check tensorflow_docs.tools.nbfmt and tensorflow_docs.tools.nblint for every commit. Please take a look in PR #1049 as well.

**Pull request merged**